Resources/Prototypes/SS220/Flavors/flavors.yml

```
@@ -0,0 +1,34 @@
+- type: flavor
+  id: shawarma
+  flavorType: Complex
+  description: Ням. На вкус как мясо со свежими овощами в лаваше.
+
+- type: flavor
+  id: borsch
+  flavorType: Complex
+  description: Ням. На вкус мясной бульон с овощами.
+
+- type: flavor
+  id: shchi
+  flavorType: Complex
+  description: Ням. На вкус кислый бульон с овощами. 
+
+- type: flavor
+  id: okroshka
+  flavorType: Complex
+  description: Ням. Освежающий вкус мяса и овощей. 
+
+- type: flavor
+  id: okroshkaKvass
+  flavorType: Complex
+  description: Ням. Освежающий вкус мяса и овощей с кислинкой.
+
+- type: flavor
+  id: vinaigrette
+  flavorType: Complex
+  description: Ням. На вкус соленые вареные овощи.
+
+- type: flavor
+  id: beets
+  flavorType: Complex
+  description: Ням. На как свекла. 
```

```suggestion
  description: Ням. На вкус сладко и как земля. 
```
прикол в том, что у свеклы землистый вкус из-за химического соединения